bindings/ios/MEGABackgroundMediaUpload.mm

```diff
@@ -0,0 +1,71 @@
+/**
+ * @file MEGABackgroundMediaUpload.mm
+ * @brief Time zone details
```

Comment:
I think this is not the brief of this class.

Reply:
It was copied across. fixed, thanks!